pulp_smash/tests/ostree/api_v2/test_copy.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Test copy of specific content from/to OSTree repositories."""
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import OSTREE_BRANCH, OSTREE_FEED, REPOSITORY_PATH,\
+    OSTREE_FEED_SMALL
```

Comment:
OSTREE_FEED_SMALL was removed, and another variable called OSTREE_BRANCHES was created. These changes will reflect the use of pulp_fixtures to generate OSTree data.
OSTree fixtures are now available. See: PulpQE/pulp-fixtures#56